JASP-Engine/JASP/R/reliabilityAnalysisFrequentist.R

```diff
@@ -404,14 +420,40 @@ reliabilityFrequentist <- function(jaspResults, dataset, options) {
 
 .frequentistReliabilityScaleTable <- function(jaspResults, model, options) {
   
-  if (!is.null(jaspResults[["scaleTableF"]])) {
+  if ((!options[["mcDonaldScalef"]] & !options[["alphaScalef"]] & !options[["guttman2Scalef"]] 
+       & !options[["guttman6Scalef"]] & !options[["glbScalef"]] & !options[["averageInterItemCorf"]]
+       & !options[["meanScalef"]] & !options[["sdScalef"]])) {
+    scaleTableF <- createJaspTable(gettext("Frequentist Scale Reliability Statistics"))
```

Comment:
Same suggestion here as for the Bayesian one

Reply:
okay, I am done with it. Can you take a look again?